infra/main.bicep

```diff
@@ -26,6 +26,8 @@ param tags object = {}
 param administratorPassword string
 @description('Specifies the resource ID of the default storage account file system for synapse.')
 param synapseDefaultStorageAccountFileSystemId string
+@description('Specifies whether an Azure Data Explorer Pool should be deployed inside your Synapse workspace as part of the template. If you selected dataFactory as processingService, leave this value as is.')
```

Comment:
Minor change - change to "Workspace" in description.

Thanks

Reply:
Let's get this done as part of another PR.